### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by mhamidjamil**
- Script will first ping if the module has package or not, it will make ttgo ready to send message if it isn't already.
- All the loggers files are now managed in a single function which need the file name and content to send.
- issue #71 was sort out by removing the redundant line `file.close()` as the **with** statement automatically takes care of closing the file when the block is exited. still need to think about #67.

### CODE REVIEW

File: temperature_work/get_temperatures.sh
Comment (wasey-rao) [inline temperature_work/get_temperatures.sh:1]: please use this in app.py as soon as possible.

**Review by wasey-rao — APPROVED** [review]

**Comment by wasey-rao** [comment]
Please complete #51 in this PR and then merge this branch, thanks.

File: temperature_work/get_temperatures.sh
Comment (mhamidjamil) [reply temperature_work/get_temperatures.sh:1]: sure

**Comment by mhamidjamil** [comment]
@wasey-rao I have resolved #51 and also implemented more features like #79 and now script will adjust its behaviour according to it executer, like if you execute it from service (default flow) then it will wait for 10 minutes so our routers powerup and if we run it directly from terminal it executes directly without any delay. Now you should merge this PR.

File: serial_communication/web/app.py [hunk 1]
```diff
@@ -92,18 +101,43 @@ def send_auth_route():
...
+def one_time_task():
+    try:
+        global boot_up_message_send
+        # Delay the execution of send_ngrok_link() by 5 minutes
+        
+        # Check if send_ngrok_link() has not been called yet
+        if not is_ngrok_link_sent():
+            time.sleep(20)
+            if not boot_up_message_send:
+                send_message("Orange Pi just boot-up time stamp: "+ fetch_current_time_online())
```
Comment (wasey-rao) [inline serial_communication/web/app.py:116]: I think it might not send boot up message @mhamidjamil please make sure it does.

**Pull request merged**